rest/route/distro.go

```diff
@@ -240,3 +289,41 @@ func (dgh *distroGetHandler) Run(ctx context.Context) gimlet.Responder {
 
 	return resp
 }
+
+////////////////////////////////////////////////////////////////////////
+
+func validateToService(ctx context.Context, apiDistro *model.APIDistro, resourceId string, isNewDistro bool) (*distro.Distro, gimlet.Responder) {
```

Comment:
nit: I might change this to "validateDistro"

I might also think about putting this in the model/data package rather than here, but it is probably alright.